common.yml

```diff
@@ -2098,87 +2196,108 @@ buildvariants:
   - ubuntu1804-power8
   stepback: false
   batchtime: 10080 # weekly
+  expansions:
+    <<: *mongod_default_startup_args
+    <<: *mongo_default_startup_args
+    mongo_os: "ubuntu1804"
+    mongo_edition: "enterprise"
+    mongo_arch: "ppc64le"
+    resmoke_use_ssl: _ssl
+    resmoke_args: -j 4
+    excludes: requires_mmap_available,requires_large_ram,requires_mongo_24,requires_mongo_26,requires_mongo_30
+    edition: enterprise
+    run_kinit: true
   tasks:
-  - name: dist
-  - name: sign
-    run_on: amazon1-2018-test
-  - name: push
-    run_on: rhel70-small
+  - name: "unit"
+  - name: ".4.2"
+  - name: ".4.4"
+  - name: "dist"
+  - name: "sign"
+    run_on: rhel80-small
+  - name: "push"
+    run_on: rhel80-small
 
 #######################################
 #     Z (s390x) Buildvariants         #
 #######################################
 
-- name: rhel67-s390x
-  display_name: ZAP s390x RHEL 6.7
+- name: rhel72-s390x
```

Comment:
you'll probably see rebase conflicts when you rebase on the lastest master, but as of TOOLS-2866 we don't support any zseries platforms (except for rhel8-zseries, which we haven't added yet)